rules/windows/process_creation/process_creation_win_manipulation_of_tokens_via_impersonate_binary.yml

```diff
@@ -0,0 +1,30 @@
+title: Impersonate Execution
+id: cf0c254b-22f1-4b2b-8221-e137b3c0af94
+status: experimental
+description: Detects execution of the Impersonate tool. Which can be used to manipulate tokens on a Windows computers remotely (PsExec/WmiExec) or interactively
+references:
+    - https://sensepost.com/blog/2022/abusing-windows-tokens-to-compromise-active-directory-without-touching-lsass/
+    - https://github.com/sensepost/impersonate
+author: Sai Prashanth Pulisetti @pulisettis
+date: 2022/12/21
+tags:
+    - attack.privilege_escalation
+    - attack.defense_evasion
+    - attack.t1134.001
+    - attack.t1134.003
+logsource:
+    product: windows
+    category: process_creation
+detection:
+    selection:
+        HostApplication|contains|all:
+            - 'impersonate.exe'
+        HostApplication|contains:
+            - 'list'
+            - 'exec'
+            - 'token'
+            - 'imp_exe'
```

Comment:
From the [source code](https://github.com/sensepost/impersonate/blob/c411d846f41647cfd2ee6fcdb00721072b283e08/Impersonate/Impersonate/Impersonate.cpp#L197-L198). We only get the following CLI options: `help`, `list`, `exec` and `adduser`

`token` and `imp_exe` do not seem available

Reply:
right, removed token and imp_exe